apps/console/src/configs/routes.tsx

```diff
@@ -30,13 +30,12 @@ import {
     UserCircleDotIcon,
     UserGroupIcon
 } from "@oxygen-ui/react-icons";
-import { APIResourcesConstants } from "@wso2is/admin.api-resources.v1/constants";
 import { getSidePanelIcons } from "@wso2is/admin.core.v1/configs/ui";
 import { AppConstants } from "@wso2is/admin.core.v1/constants";
 import { commonConfig } from "@wso2is/admin.extensions.v1";
 import { FeatureGateConstants } from "@wso2is/admin.extensions.v1/components/feature-gate/constants/feature-gate";
 import { ServerConfigurationsConstants } from "@wso2is/admin.server-configurations.v1";
-import { LegacyModeInterface, RouteInterface } from "@wso2is/core/models";
+import { LegacyModeInterface, RouteInterface } from "@wso2is/core/src/models";
```

Comment:
Is the `src` part expected?

Reply:
Fixed with [d0f8fab](https://github.com/wso2/identity-apps/pull/6487/commits/d0f8fab615c67a855bbf44a0fb3bc7a77aae9c94)